libs/utils/utils.js

```diff
@@ -620,6 +620,23 @@ export function decorateAutoBlock(a) {
   });
 }
 
+const decorateCopyLink = (a, evt) => {
+  const userAgent = navigator.userAgent.toLowerCase();
+  const isMobile = /android|iphone|ipod|blackberry|iemobile|opera mini|mobile/.test(userAgent) && !/ipad/.test(userAgent);
```

Comment:
I really don't think we need to worry about `blackberry`, `iemobile` and `opera mini`

@overmyheadandbody @narcis-radu Thoughts?

Reply:
I agree, good observation! I'd also argue that `ipod` is not needed. Per [Wikipedia](https://en.wikipedia.org/wiki/IPod), iPods have been discontinued on May 10, 2022. Looking at our RUM data, I couldn't find users navigating from iPods.